internal/monitors/collectd/custom/metadata.yaml

```diff
@@ -43,6 +43,22 @@ monitors:
     directory, where `<AGENT_BUNDLE>` is the root of the filesystem in the
     containerized version, and is normally `/usr/lib/signalfx-agent` in the
     non-containerized agent.
+
+    ### Running the collectd/exec plugin
+    It is possible to use the collectd/custom monitor to run the collectd/exec
```

Comment:
You can use the collectd/custom monitor to run the collectd/exec plugin. If you are running the Smart Agent in a container, then use the 'signalfx-agent' user when you run the following script:

To learn how to use the plugin, see ...........